src/pages/Transaction/TransactionCellList/index.tsx

```diff
@@ -20,11 +20,15 @@ export default ({
   outputs,
   txHash,
   showReward,
+  total,
+  indiceOffset,
 }: {
   inputs?: Cell[]
   outputs?: Cell[]
   txHash?: string
   showReward?: boolean
+  total?: number
+  indiceOffset?: number
```

Comment:
How about loading `offset` from the url, the `page_number` and `page_size` parameters, and get the offset by `(page_number - 1) * page_size`

Reply:
There are `input_cells_page_size/number` and `output_page_size/number` params, if `offset` is removed, then need to add another field like `cell_input_output_type` to distinguish them

Reply:
In what case will the sizes of input cells and output cells be different? I didn't find an entry to update the size on the page.

Reply:
[43894e5](https://github.com/Magickbase/ckb-explorer-frontend/pull/188/commits/43894e590acfaeb929997dcd42556a4362944e69) has added page size change for pagination component, but there is no design yet, I may change the appearance according to design, please have a look @Sven-TBD 
<img width="800" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/3870972/02c7fda6-bcd4-4048-bd27-10f40d53b9f6">

Reply:
Ya, we could add a drop box with some default options for the user @Kirl70 , like 5,10,15.

Reply:
Design Draft:
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=6102%3A26855&mode=design&t=tPSt89MOl6FwQfR3-1
<img width="820" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/106060898/3681a1c4-dd44-4396-b6dd-7da4bd3a7ce4">

Reply:
I found the `page size` component exists in `input list` and `output list`, so their `size`s are independent, am I right? @Sven-TBD